components/SecondaryAppbar.tsx

```diff
@@ -0,0 +1,44 @@
+import { GestureResponderEvent, StyleSheet } from 'react-native';
+import {
+  Appbar as PaperAppbar,
+  IconButton,
+  useTheme,
+} from 'react-native-paper';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+const styles = StyleSheet.create({
+  bar: {
+    height: 'auto',
+    shadowColor: 'blue',
+    shadowOffset: { width: 2, height: 2 },
+    shadowOpacity: 0.9,
+    shadowRadius: 5,
+    elevation: 10,
+  },
+});
+
+export type SecondaryAppbarProps = {
+  onBackPressed: (e: GestureResponderEvent) => void;
+};
+
+export const SecondaryAppbar = (props: SecondaryAppbarProps) => {
```

Comment:
we should add some parameters to the existing `AppBar` and customise it rather than creating a new component.
e.g. `forDetailScreen` or something similar.

Reply:
done. Added `forDetailScreen` parameter.